### ISSUE TITLE
Scaling keypoints

### ISSUE THREAD

**Issue by mukeshnarendran7** [BlurryLight/DD-Net-Pytorch#3]
Thanks for the great repo. Id like to try to apply it to a custom dataset. 
I noticed that the keypoints in the JHMDB dataset files are in a different scale. How can i normalize my keypoints similarly before passing it in through the dataloader?

**Comment by BlurryLight** [comment]
I have been away from this field for a long time so I don't remember details.
